package.json

```diff
@@ -27,7 +27,8 @@
     "eslint": "^8.23.0",
     "ganache": "^7.4.4",
     "solhint": "^3.3.7",
-    "truffle": "^5.6.0"
+    "truffle": "^5.6.0",
+    "bytes32": "^0.0.3"
```

Comment:
Do we need this?

Reply:
There is still more work to be done on tests, this should help with crafting values, tests with its usage will be pushed with next commit. Don't focus too much on tests yet.